drivers/extend/gpio_ext.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2018 Acutam Automation, LLC
+ *               2019 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_extend_gpio
+ *
+ * @{
+ *
+ * @file
+ * @brief       GPIO extension functions
+ *
+ * @author      Matthew Blue <matthew.blue.neuro@gmail.com>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#if MODULE_EXTEND_GPIO
+
+#include "periph/gpio.h"
+#include "extend/gpio.h"
+#include "gpio_ext_conf.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#endif /* MODULE_EXTEND_GPIO */
```

Comment:
I haven't checked the `#include`ed headers yet, but is this seems to be empty. Is this file needed?

If I recall correctly, the C standard forbids an empty complication unit. So you should add `typedef int dont_be_pedantic;` at least in the case `MODULE_EXTEND_GPIO` is not defined to have the file confirming with the standard. (Even though both GCC and clang don't care about empty compilation units, I believe there is no harm in being strictly compliant with the standard.)

Reply:
I do not remember, but there must have been a reason why I added this empty `.c` file. Maybe it had something to do with the order in which `gpio_ext_conf.h` was included. I will check it again.

Reply:
I think the file could be removed.